src/graphql/data/util.ts

```diff
@@ -12,6 +12,8 @@ export enum TimePeriod {
   ALL,
 }
 
+export type TimePeriodWithoutAll = Exclude<TimePeriod, TimePeriod.ALL>
```

Comment:
Why not just remove ALL from TimePeriod for now? That seems less bug-prone than having two nearly-compatible enums be exported.

Reply:
makes sense done